app/api/server/v1/teams.ts

```diff
@@ -151,7 +151,7 @@ API.v1.addRoute('teams.members', { authRequired: true }, {
 		const { offset, count } = this.getPaginationItems();
 		const { teamId, teamName } = this.queryParams;
 
-		const { records, total } = Promise.await(Team.members(this.userId, teamId, teamName, { offset, count }));
+		const { records, total } = Promise.await(Team.members(teamId, teamName, { offset, count }));
```

Comment:
This change may not be compatible with other calls to `members`, please check if the change is compatible